src/protocols/secure_channel/Constants.h

```diff
@@ -63,10 +63,12 @@ enum class MsgType : uint8_t
     PASE_PakeError     = 0x2F,
 
     // Certificate-based session establishment Message Types
-    CASE_SigmaR1  = 0x30,
-    CASE_SigmaR2  = 0x31,
-    CASE_SigmaR3  = 0x32,
-    CASE_SigmaErr = 0x3F,
+    CASE_Sigma1       = 0x30,
+    CASE_Sigma2       = 0x31,
+    CASE_Sigma3       = 0x32,
+    CASE_Sigma1Resume = 0x33,
```

Comment:
Sigma1Resume is no longer use. We always use CASE_Sigma1. Will clarify the spec editorially. The table still havs both Sigma1Resume/SigmaErr in the SecureChannel section due to an omission.

Reply:
See https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/4183